examples/scripts/ct_astra_tv_admm.py

```diff
@@ -5,16 +5,14 @@
 # with the package.
 
 r"""
-Few-View CT (ADMM w/ Total Variation)
-=====================================
+TV Regularized Few-View CT Reconstruction
```

Comment:
Is few-view the same as sparse-view? I never heard it before.

Reply:
Changed `few-view` to `sparse-view`.